src/test/java/org/fundacionjala/coding/Convert/DescendTest.java

```diff
@@ -0,0 +1,38 @@
+package org.fundacionjala.coding.Convert;
+import org.junit.Assert;
+
+/**
+ * Test to Descend convertor.
+ */
+
+public class DescendTest {
+
+  /**
+   * test 1.
+   */
+  @org.junit.Test
+  public void sortedWordsDescend1() {
```

Comment:
Your implementation is failing for the following scenario.
```java
    String text = "I am";
    Descend descend = new Descend(text);
    descend.sortedWordsDescend();

    String textExpected = "I am ";
    Assert.assertEquals(textExpected,descend.getResult());
```

Reply:
got it , reviewing thanks